portfolio/src/main/webapp/script.js

```diff
@@ -22,6 +22,33 @@ function fetchData() {
   });
 }
 
+function fetchComments() {
+  fetch('/data').then(response => response.json()).then((commentData) => {
+    
+    //Test console log statements
+    console.log("Hi")
+    console.log(commentData);
+    
+    //Loops through elements of json object to get every comment
```

Comment:
The loop below is fairly easy to figure this out by looking at it. Consider dropping this comment.